src/CasCap.Common.Threading/Logic/AsyncDuplicateLock.cs

```diff
@@ -3,6 +3,7 @@
 /// <summary>
 /// asynchronous locking based on a key
 /// </summary>
+[Obsolete("Use AsyncKeyedLock package instead.")]
```

Comment:
Deprecated via use of AsyncKeyedLock, https://www.nuget.org/packages/AsyncKeyedLock

Reply:
Done.